recipes/mne-qt-browser/meta.yaml

```
@@ -0,0 +1,51 @@
+{% set name = "mne-qt-browser" %}
+{% set version = "0.1.3" %}
```

```suggestion
{% set version = "0.1.4" %}
```

I increased the version-number once more yesterday for the newest feedback on the now merged PR in MNE-Python.